src/types/tmdb/tv.ts

```diff
@@ -65,7 +66,23 @@ export interface TvDetailsData extends TvData {
         vote_count: number;
     };
     media_type: 'tv';
-    next_episode_to_air: null;
+    next_episode_to_air:
+        | {
```

Comment:
Is this first pipe character supposed to be there?

Reply:
See below